client/store/user.js

```diff
@@ -56,9 +58,9 @@ export const logout = () => async dispatch => {
   }
 }
 
-export const updateUser = (user, userId) => async dispatch => {
-  const currentUser = await axios.put(`/api/users/${userId}/profile`, user)
-  dispatch(getUser(currentUser.data))
+export const updateUser = (editedUser, userId) => async dispatch => {
```

Comment:
I'm pretty sure I wrote this portion... but it might be worth spreading `{...user, userId}` into one object?

Reply:
Yes, this is an ideal scenario. However, I ran into a database error when trying to pass the full user state-object into the thunk. At some point, a property, "events" is being appended to our user state-object, and our instance cannot be updated because "events" is not a column in "user". I have absolutely no idea why we're not encountering this issue with "skillIds", which is something that I'm appending (doing all of this, by the way, is definitely an anti-pattern when using Redux. ONLY Redux should be allowed to "mutate" the state object... you and I both know that it's not actually mutating it, but creating a new state object with the changes). Instead, the EDIT_USER action is only accepting changed values and merging them into the existing state object. Since we're not passing a full user state-object, we need to provide a user's id as a separate argument.